### ISSUE TITLE
Feature: npc_egg

### ISSUE THREAD

**Issue by Godohwar** [ChaosInitiative/Portal-2-Community-Edition#714]
### Which component should be improved?

Entity

### Describe your suggestion

P2CE is an amazing mod but I was devastated to find out that it is missing the crucial entity "npc_egg".

The gravity that this mod doesn't have an npc_egg entity absolutely crushed me and I knew I had to make this issue.

It is absolutely crucial to the development of P2CE that such entity "npc_egg" is added to the mod, and without it the mod will be much more dull.

What is the function of the entity "npc_egg" you might question? It functions like an egg does in real life. It is a tangible object that is spherical in shape and ranges in colors from white to tan. The egg can be picked up, thrown, and can be cracked. Eggs are also capable of military assault with the usage of such guns as an AK-47. I will be bringing up the addition of an AK-47 at a later date.

In conclusion, npc_egg should be added to P2CE at all costs.

### Expected result

When I place the entity "npc_egg" it should be able to use any firearms given to it, or be turned into an omelet.

**Comment by craftablescience** [comment]
This can be added through ChaosScript at a later date.

**Comment by CitadelCore** [comment]
Please don't use this issue tracker to shitpost.
